components/event-publisher-proxy/pkg/metrics/collector.go

```diff
@@ -64,3 +78,8 @@ func (c *Collector) RecordError() {
 func (c *Collector) RecordLatency(duration time.Duration) {
 	c.latency.WithLabelValues().Observe(duration.Seconds())
 }
+
+// RecordEventType records a eventType metric
+func (c *Collector) RecordEventType(eventType, eventSource string) {
+	c.eventType.WithLabelValues(eventType, eventSource).Inc()
```

Comment:
What happens to the metric EventType when EPP is restarted/crashed.
Does it starts counting from zero again?